### PR TITLE
Add K5 draft contained levels to Redshift information.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Per request from @kiki-lee.

### CODE REVIEW

File: bin/cron/create_rollup_tables [hunk 1]
```diff
@@ -86,6 +86,32 @@ def main
...
+  Level.find_each do |level|
+    next unless level.script_levels.map(&:script).select(&:k5_draft_course?)
+
+    level.contained_levels.each do |contained_level|
+      contained_question = get_contained_question(contained_level)
+      contained_levels << {
+        level_id: level.id,
+        contained_level_id: contained_level.id,
+        contained_level_type: contained_level.type,
+        contained_level_page: page_index,
```
Comment (Hamms) [inline bin/cron/create_rollup_tables]: Shouldn't this be `level.page_index`? Or am I misunderstanding something?

Comment (Hamms) [reply bin/cron/create_rollup_tables]: Or `contained_level.page_index`?

File: bin/cron/create_rollup_tables [hunk 2]
```diff
@@ -86,6 +86,32 @@ def main
...
+        contained_level_text: sanitize_string_for_db(contained_question)
+      }
+
+      answers = contained_level.properties['answers']
+      answers.each_with_index do |answer, answer_number|
+        contained_level_answers << {
+          contained_level_id: contained_level.id,
+          answer_number: answer_number,
+          answer_text: sanitize_string_for_db(answer['text']),
+          correct: answer['correct']
```
Comment (Hamms) [inline bin/cron/create_rollup_tables:109]: Why doesn't this need to be sanitized?

File: bin/cron/create_rollup_tables (hunk 1 above)
Comment (ashercodeorg) [reply bin/cron/create_rollup_tables]: Nice catch, the `page_index` should be `1` and the `position_index` should be the `contained_level` index. Done.

File: bin/cron/create_rollup_tables (hunk 2 above)
Comment (ashercodeorg) [reply bin/cron/create_rollup_tables:109]: It should be a boolean. E.g.,
```
Level.find_by_id(8776).contained_levels.first.properties['answers'][0]['correct']
=> false
```

File: bin/cron/create_rollup_tables (hunk 1 above)
Comment (ashercodeorg) [reply bin/cron/create_rollup_tables]: Note that the former assumes there is no pagination, which I'm pretty sure is 100% safe.

**Comment by ashercodeorg** [comment]
PTAL.

**Comment by ashercodeorg** [comment]
Circle CI failure is known flakiness.

**Pull request merged**